components/tab-panel/index.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import { partial, noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+// import './style.scss';
+import { Component } from '@wordpress/element';
+import { default as withInstanceId } from '../higher-order/with-instance-id';
+import { NavigableMenu } from '../navigable-container';
+
+class TabButton extends Component {
+	render() {
+		const { tabId, children, clickTab, selected, ...rest } = this.props;
+		return <button role="tab"
+			tabIndex={ selected ? null : -1 }
+			aria-selected={ selected }
+			id={ tabId }
+			onClick={ partial( clickTab, tabId ) }
+			{ ...rest }>
```

Comment:
minor but for multiline elements like this, I'd prefer something like:

```js
return (
   <element
      prop="value"
      prop2="value2"
   >
      children
   </element>
);
```

Reply:
Good. I find that more readable :)